integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+                'source': u'openstreetmap.org',
+                'to': u'Gevgelija',
+                'type': u'route',
+            }, ways=[461227650]),
+            dsl.relation(2, {
+                'name': u'European Route 75',
+                'network': u'e-road',
+                'ref': u'E 75',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikipedia': u'en:European route E75',
+            }, ways=[461227650]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 461227650,
+                'network': u'MK:national',
+            })
+
+    def test_myfederal(self):
+        import dsl
+
+        z, x, y = (16, 51652, 32357)
+
+        self.generate_fixtures(
+            dsl.is_in('MY', z, x, y),
+            # https://www.openstreetmap.org/way/553323706
+            dsl.way(553323706, dsl.tile_diagonal(z, x, y), {
+                'highway': u'trunk',
+                'name': u'Jalan Kluang-Jemaluang',
+                'oneway': u'no',
+                'ref': u'50',
+                'source': u'openstreetmap.org',
+                'surface': u'paved',
+            }),
+            dsl.relation(1, {
+                'distance': u'135.38',
+                'name': u'Jalan Batu Pahat - Ayer Hitam - Jemaluang',
+                'network': u'my:federal',
+                'ref': u'50',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikipedia': u'en:Malaysia Federal Route 50',
+            }, ways=[553323706]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 553323706,
+                'network': u'MY:federal',
+            })
+
+    def test_nofylkesvei(self):
+        import dsl
+
+        z, x, y = (16, 33706, 18951)
+
+        self.generate_fixtures(
+            dsl.is_in('NO', z, x, y),
+            # https://www.openstreetmap.org/way/106337203
+            dsl.way(106337203, dsl.tile_diagonal(z, x, y), {
+                'highway': u'primary',
+                'maxspeed': u'50',
+                'ref': u'555',
+                'source': u'openstreetmap.org',
+                'surface': u'asphalt',
+            }),
+            dsl.relation(1, {
+                'name': u'Fylkesvei 555 (Hordaland)',
+                'network': u'no:Fylkesvei',
+                'operator': u'Hordaland Fylkeskommune',
+                'ref': u'555',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q6367615',
+                'wikipedia': u'no:Fylkesvei 555',
+            }, ways=[106337203]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 106337203,
+                'network': u'NO:Fylkesvei',
```

Comment:
What's our convention on initial cap on `AA:bb` bb portion? Is it always data driven or do we force it up or down?

Reply:
I've mostly kept it close to what's in the data, unless there seemed to be a reason to change it. We can easily force it to lower case, if that's what you'd prefer?